app/models/series.rb

```diff
@@ -9,8 +9,6 @@ class Series < ActiveFedora::Base
   validates :title, presence: { message: 'Your series must have a title.' }
   validates :description, presence: { message: 'Your series must have a description.' }
 
-  self.human_readable_type = 'Series'
-
```

Comment:
@adeelahmad Do you know what `.human_readable_type` got replaced with, if anything?

Reply:
@afred its being read by locale files